vnc.html

```diff
@@ -297,12 +297,16 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br>VNC</h1>
     <div class="noVNC_center noVNC_connect_layer">
     <div id="noVNC_password_dlg" class="noVNC_panel"><form>
```

Comment:
Maybe the name of this container div should be changed to noVNC_credentials_dlg to be in line with your other name changes